test/server/cards/01_SOR/bases/SecurityComplex.spec.js

```diff
@@ -0,0 +1,34 @@
+describe('SecurityComplex', function() {
+    integration(function() {
+        describe('Security Complex\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        base: 'security-complex',
```

Comment:
We should add at least one unit on the friendly side to confirm that the ability can target units controlled by either player